bindings/nodejs/lib/types/block/output/token-scheme.ts

```diff
@@ -26,15 +27,11 @@ abstract class TokenScheme {
  * Simple token scheme.
  */
 class SimpleTokenScheme extends TokenScheme {
-    private mintedTokens: bigint;
-    private meltedTokens: bigint;
-    private maximumSupply: bigint;
+    private mintedTokens: u64;
+    private meltedTokens: u64;
+    private maximumSupply: u64;
```

Comment:
These are U256

Reply:
Also the other native token things below

Reply:
shit didnt even think of that madness

Reply:
fixed